dotnet/src/Connectors/Connectors.Amazon/Bedrock/Settings/AmazonClaudeExecutionSettings.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon;
+
+/// <summary>
+/// Prompt execution settings for Anthropic Claude Text Generation
+/// </summary>
+[JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
+public class AmazonClaudeExecutionSettings : PromptExecutionSettings
```

Comment:
@charlizeyeh 
For the next PR.

Currently all our connectors that targets `Text` and `Chat` modalities ends up with `PromptExecutionSettings` suffix, after a internal conversation we are going to keep for now that suffix. 

Moving forward, suggest updating the classes to:

`Amazon{Model}PromptExecutionSettings`.

Thanks.